ui/app/components/clients/attribution.js

```diff
@@ -103,9 +104,24 @@ export default class Attribution extends Component {
     }
   }
 
-  get getCsvData() {
+  destructureCountsToArray(object) {
+    let { clients, entity_clients, non_entity_clients } = object;
+    return [clients, entity_clients, non_entity_clients];
+  }
+
+  constructCsvRow(firstColumn, secondColumn = null, otherColumns) {
+    // ['ns label', 'mount label', 'total clients', 'entity', 'non-entity']
+    // if constructing namespace row, mount column will be blank
```

Comment:
maybe ... "_the_ mount _label_ column will be blank. On first read I was expecting there to be a column named mount.